aiida_registry/fetch_metadata.py

```diff
@@ -47,9 +49,42 @@ def get_hosted_on(url):
 
     return netloc
 
+def get_github_commits_count(repo_url, token):
+    owner, repo = repo_url.split('/')[-2:]
+    url = f"https://api.github.com/repos/{owner}/{repo}/commits"
```

Comment:
Maybe worth adding a comment here that this counts the commits on the default branch of the repository (is this correct?).
I think this is also what we want, but still good to mention (same for the git-based approach).

Reply:
Yes, this is correct. Updated.